wikitemplate-IPFS.md

```diff
@@ -167,30 +204,26 @@
 
 - [ ] Prerequisites: local node launched and local gateway configured. Go to `Settings -> IPFS`, there should be an available `Set up your IPNS keys` option, which opens `brave://settings/ipfs/keys`
 
-   ### `Sharing a local file using an IPFS key`
+   ### `Sharing a local file using an IPNS key`
 
-   - [ ] Verify you're able to share a local file via `IPFS > Share local file using IPFS`.
+   - [ ] Verify you're able to share a local file via `IPFS -> Share local file using IPNS -> self`.
```

Comment:
This assumes UI change from https://github.com/brave/brave-browser/issues/16998 happens.

Reply:
Thanks; in that case, would it make sense to add a note indicating that, so we can track and remove later once landed?